Tools/freeze/freeze.py

```diff
@@ -217,7 +217,12 @@ def main():
 
     # locations derived from options
     version = '%d.%d' % sys.version_info[:2]
-    flagged_version = version + sys.abiflags
+    if sys.platform != "win32":
```

Comment:
There's a variable defined called `win`, so I believe this check can be changed to check `if not win:`